Engine/source/gfx/screenshot.cpp

```diff
@@ -130,85 +130,85 @@ void ScreenShot::capture( GuiCanvas *canvas )
+
+			//// The current GFX device couldn't capture the backbuffer or it's unable of doing so.
+			//if (gb == NULL)
+			//	return;
+
+   //               
+   //      // Copy the captured bitmap into its tile
+   //      // within the output bitmap.         
+   //      const U32 inStride = gb->getWidth() * gb->getBytesPerPixel();         
+   //      const U8 *inColor = gb->getBits() + inStride * overlapPixels.y;
+   //      const U32 outStride = outBuffer->getWidth() * outBuffer->getBytesPerPixel();
+   //      const U32 inOverlapOffset = overlapPixels.x * gb->getBytesPerPixel();         
+   //      const U32 inOverlapStride = overlapPixels.x * gb->getBytesPerPixel()*2;
+   //      const U32 outOffset = (tx * (gb->getWidth() - overlapPixels.x*2 )) * gb->getBytesPerPixel();
+   //      U8 *outColor = outBuffer->getWritableBits() + outOffset;
+   //      for( U32 row=0; row < gb->getHeight() - overlapPixels.y; row++ )
+   //      {
+   //         dMemcpy( outColor, inColor + inOverlapOffset, inStride - inOverlapStride );
+   //         
+   //         //Grandient blend the left overlap area of this tile over the previous tile left border
+   //         if (tx && !(ty && row < overlapPixels.y))
+   //         {
+   //            U8 *blendOverlapSrc = (U8*)inColor;
+   //            U8 *blendOverlapDst = outColor - inOverlapOffset;
+   //            for ( U32 px=0; px < overlapPixels.x; px++)
+   //            {
+   //               F32 blendFactor = (F32)px / (F32)overlapPixels.x;
+   //               sBlendPixelRGB888(blendOverlapSrc, blendOverlapDst, blendFactor);                 
+
+   //               blendOverlapSrc += gb->getBytesPerPixel();
+   //               blendOverlapDst += outBuffer->getBytesPerPixel();                   
+   //            }               
+   //         }
+
+   //         //Gradient blend against the rows the excess overlap rows already in the buffer            
+   //         if (ty && row < overlapPixels.y)
+   //         {
+   //            F32 rowBlendFactor = (F32)row / (F32)overlapPixels.y;
+   //            U8 *blendSrc = outColor + outStride * (outBuffer->getHeight() - overlapPixels.y);
+   //            U8 *blendDst = outColor;               
+   //            for ( U32 px=0; px < gb->getWidth() - overlapPixels.x*2; px++)
+   //            {                  
+   //               sBlendPixelRGB888(blendSrc, blendDst, 1.0-rowBlendFactor); 
+   //               blendSrc += gb->getBytesPerPixel();
+   //               blendDst += outBuffer->getBytesPerPixel();                   
+   //            }                              
+   //         }
+
+   //         
+   //         inColor += inStride;
+   //         outColor += outStride;
+   //      }
+
+   //      delete gb;
+   //   }
+
+   //   // Write the captured tile row into the PNG stream
+   //   pngWriter.append(outBuffer, outBuffer->getHeight()-overlapPixels.y);
+   //}
+
+   ////Close the PNG stream
+   //pngWriter.end();
```

Comment:
Please learn to:
```cpp
#if 0
// code to disable goes here
#endif // 0
```

Instead of multi-line commenting out block of code to avoid these unnecessary large diffs.

Reply:
> Please learn to:
> 
> ```c++
> #if 0
> // code to disable goes here
> #endif // 0
> ```
> 
> Instead of multi-line commenting out block of code to avoid these unnecessary large diffs.

was talked over with az and jeff, this is just commented out for now to be changed when we figure out how to do this with stb image